src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/medical/Medicine.java

```diff
@@ -21,6 +22,7 @@ public ItemConsumptionHandler getItemHandler() {
         return (e, p, item) -> {
             p.setFireTicks(0);
             clearNegativeEffects(p);
+            RadioactivityTask.removePlayer(e.getPlayer());
```

Comment:
Either move this into the `clearNegativeEffects` or at least use the `p` variable.